perf/tbench.py

```diff
@@ -60,7 +60,7 @@ def setUp(self):
     def test(self):
         # only supports combined server+client model at the moment
         # should support separate I suppose, but nobody uses it
-        nprocs = self.params.get('nprocs', default=commands.getoutput("nproc"))
+        nprocs = self.params.get('nprocs', default=subprocess.getoutput("nproc"))
```

Comment:
@shirishaganta Can you import cpu util and use `cpu.online_count` or `cpu.total_count` whichever suits best?

Reply:
okay